build/Microsoft.DotNet.Cli.Prepare.targets

```diff
@@ -46,6 +47,10 @@
     <PropertyGroup>
       <VersionBadge>$(BaseOutputDirectory)/$(VersionBadgeMoniker)_$(Configuration)_version_badge.svg</VersionBadge>
       <SdkVersion>$(NugetVersion)</SdkVersion>
+      
+      <SdkProjectJsonVersion>$(VersionMajor).$(VersionMinor).$(VersionPatch)-preview3-$(CommitCount)</SdkProjectJsonVersion>
```

Comment:
I don't think "ProjectJsonVersion" is appropriate.  This should be "NuGetVersion" or something that is more long-term than "ProjectJson", which should be going away soon.